src/components/CorporateCampaign/CampaignVerificationForm.vue

```diff
@@ -9,7 +9,7 @@
 					:width="iFrameWidth"
 					frameborder="0"
 				></iframe>
-				<script src="//kiva.tfaforms.net/js/iframe_resize_helper.js"></script>
```

Comment:
oh nice, to get my local working today I removed this from the html and added it to metaInfo
```
	metaInfo: {
		script: [
			{ src: '//kiva.tfaforms.net/js/iframe_resize_helper.js', async: true }
		],
	},
```

Not sure if there's a preferred way or not

Reply:
@ryan-ludwig can you verify if my fix works for you (I'm assuming you're on node 12, npm 6, storybook 6.x)

Reply:
I'll test this change out locally and update the PR I have just pushed that uses this if all is well.